{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tasks.py

```diff
@@ -1,8 +1,5 @@
-from django.contrib.auth import get_user_model
-
 from config import celery_app
-
-User = get_user_model()
+from {{ cookiecutter.project_slug }}.users.models import User
 
 
 @celery_app.task()
```

Comment:
Per [celery docs](https://docs.celeryq.dev/en/stable/userguide/tasks.html#how-do-i-import-the-task-decorator), 

> If you’re using Django... then you probably want to use the shared_task() decorator

Reply:
I can open a new pr and write a fix if you have no objections

Reply:
I had the vague impression that it was for reusable apps, but that was from a long time ago. I'm happy to follow the latest recommendations on this, yes. Agree that I'd rather be in a separate PR